experimental/telemetry/doc.go

```diff
@@ -15,4 +15,30 @@
 // If you want to use your own data plane URL, add also this line and
 // make sure the MM_RUDDER_DATA_PLANE_URL environment variable is set.
 //   LDFLAGS += -X "github.com/mattermost/mattermost-plugin-api/experimental/telemetry.rudderDataPlaneURL=$(MM_RUDDER_DATA_PLANE_URL)"
+//
+// In order to use it you should:
+// 1. Start the telemetry client on plugin activate:
+//  func (p *Plugin) OnActivate() error {
+//    p.telemetryClient, err = telemetry.NewRudderClient()
+//	  if err != nil {
+//	    p.API.LogWarn("telemetry client not started", "error", err.Error())
+//	  }
+//  }
+// 2. Init and update the tracker on configuration change
+//  func (p *Plugin) OnConfigurationChange() error {
+//    ...
+//    p.tracker = telemetry.NewTracker(p.telemetryClient, p.API.GetDiagnosticId(), p.API.GetServerVersion(), manifest.Id,
+//      manifest.Version, "pluginName", enableDiagnostics, logger)
```

Comment:
Can this be
```suggestion
//      manifest.Version, manifest.ID, enableDiagnostics, logger)
```
?

Reply:
The people from analytics preferred to have a "shorter" name for the plugins, not the id, specially since they did not specially liked `.`s in the name.

Reply:
Got it. Maybe add a comment about that?

Reply:
There is a comment about that in tracker.go.